homeassistant/components/tuya/climate.py

```diff
@@ -200,8 +200,8 @@ def __init__(
         # it to define min, max & step temperatures
         if self._set_temperature:
             self._attr_supported_features |= ClimateEntityFeature.TARGET_TEMPERATURE
-            self._attr_max_temp = self._set_temperature.max_scaled
-            self._attr_min_temp = self._set_temperature.min_scaled
+            self._attr_max_temp = self._set_temperature.max_scaled * 5
+            self._attr_min_temp = self._set_temperature.min_scaled * 5
```

Comment:
This seems wrong, could you point me to the documentation of Tuya that explains this change?

Reply:
Hello frenck, thank you for reviewing.
I didn't find this on documentation, just noticed that the climate integration reported 5 times smaller temperature than the one on device screen of my thermostat. This is reported by many users on issue #85783, since January.
I'm aware that this is not probably the best and cleanest way to do it, but it fixed the problem on my case and I would like to contribute with this.

Reply:
> just noticed that the climate integration reported 5 times smaller temperature

That is a known issue with some thermostat, that is caused by the manufacturer not using the standards as set out by Tuya correctly.

This change will therefore break all thermostats that are correctly implemented according to standards.


../Frenck